### PR TITLE
Add eulerline property to Triangle

### PR DESCRIPTION
**Pull request by jagannath93**

### CODE REVIEW

File: sympy/geometry/polygon.py [hunk 1]
```diff
@@ -2215,6 +2215,37 @@ def medial(self):
...
+
+        The line which passes through circumcenter, orthocenter and the median
+        of a triangle.
+
+        Returns
+        =======
+
+        eulerline : Line if triangle is isoceles or right angled
+
+        common center: Point if the triangle is equilateral in which case all the all
```
Comment (smichr) [inline sympy/geometry/polygon.py]: ~~all the~~

File: sympy/geometry/polygon.py [hunk 2]
```diff
@@ -2215,6 +2215,37 @@ def medial(self):
         s = self.sides
         return Triangle(s[0].midpoint, s[1].midpoint, s[2].midpoint)
 
+    @property
+    def eulerline(self):
+        """The euler line of the triangle.
```
Comment (smichr) [inline sympy/geometry/polygon.py]: euler -> Euler

File: sympy/geometry/polygon.py [hunk 3]
```diff
@@ -2215,6 +2215,37 @@ def medial(self):
...
+
+        The line which passes through circumcenter, orthocenter and the median
+        of a triangle.
+
+        Returns
+        =======
+
+        eulerline : Line if triangle is isoceles or right angled
```
Comment (smichr) [inline sympy/geometry/polygon.py]: isoceles  -> isosceles

File: sympy/geometry/polygon.py (hunk 1 above)
Comment (smichr) [inline sympy/geometry/polygon.py]: ~~at the same place~~

**Comment by jagannath93** [comment]
Hi @smichr , I've done all the corrections that you had mentioned.

File: sympy/geometry/tests/test_geometry.py [hunk 4]
```diff
@@ -1436,6 +1436,12 @@ def test_polygon():
     '''20151573611150265741278060334545897615974257/16073686192120448448157'''
     '''8148466200000000000)''')
 
+    # Eulerline
+    print(t1.eulerline)
```
Comment (debugger22) [inline sympy/geometry/tests/test_geometry.py]: I think this `print` is not required.

File: sympy/geometry/tests/test_geometry.py [hunk 5]
```diff
@@ -1436,6 +1436,12 @@ def test_polygon():
...
+    assert t1.eulerline in (Line(Point(5/3, 5/3), Point(5/2, 5/2)), 
+                            Line(Point(0, 0), Point(5/3, 5/3)),
+                            Line(Point(0, 0), Point(5/2, 5/2)))
+
```
Comment (debugger22) [inline sympy/geometry/tests/test_geometry.py]: You should also add tests when it returns a `Point`.

**Comment by gxyd** [comment]
I think we can close this. `eulerline` method was implemented in PR https://github.com/sympy/sympy/pull/10615 . Closing this.

**Pull request closed without merging**